cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -57,15 +58,21 @@ const kubernetesSvcName = "kubernetes"
 var (
 	argDomain              = flag.String("domain", "cluster.local", "domain under which to create names")
 	argEtcdMutationTimeout = flag.Duration("etcd-mutation-timeout", 10*time.Second, "crash after retrying etcd mutation for a specified duration")
-	argEtcdServer          = flag.String("etcd-server", "http://127.0.0.1:4001", "URL to etcd server")
+	argEtcdServers         = flag.StringSlice("etcd-servers", []string{"http://127.0.0.1:4001"}, "List of etcd servers to watch (http://ip:port), comma separated")
+	argEtcdCAFile          = flag.String("etcd-cafile", "", "SSL Certificate Authority file used to secure etcd communication")
+	argEtcdCertFile        = flag.String("etcd-certfile", "", "SSL certification file used to secure etcd communication")
```

Comment:
Mostly for my own edification, is this for client -> etcd or etcd <-> peer, or both? 

Also what is the format, pem encoded x509 I'm guessing? or if it's "whatever etcd accepts" then please add a comment with a link to the specification. 

What's the name in this subject, ip of client?